firefox-ios/Client.xcodeproj/project.pbxproj

```diff
@@ -22511,6 +22511,379 @@
 			};
 			name = FirefoxBeta;
 		};
+		8A19936D2C9B5A63001F4D57 /* Fennec_Testing */ = {
+			isa = XCBuildConfiguration;
+			baseConfigurationReference = E60961861B62B8A700DD640F /* Fennec.xcconfig */;
+			buildSettings = {
+				GCC_TREAT_WARNINGS_AS_ERRORS = YES;
+				OTHER_LDFLAGS = (
+					"$(inherited)",
+					"-Xlinker",
+					"-no_application_extension",
+				);
+				OTHER_SWIFT_FLAGS = "$(OTHER_SWIFT_FLAGS_common) -DMOZ_CHANNEL_FENNEC";
+				SWIFT_ACTIVE_COMPILATION_CONDITIONS = TESTING;
```

Comment:
xcode hides this file in github, but these changes are important, please verify as well!~ thank you 🙇‍♀️

Reply:
No problem. I would put this in the `FennecTesting.xcconfig` file, so is more easily findable

Reply:
When I first tried to add this, I felt I was making additional changes that were not as necessary so I leaned towards making minimal changes in the PR to keep it simple. If you think its worth having to add it, happy to make the modifications! I'll create a commit so we can see if Bitrise also likes it.

Reply:
Commit here: https://github.com/mozilla-mobile/firefox-ios/pull/22070/commits/0ee91ddc7397c628238ddcbce8a192d01c2e909f